indra/newview/llfloaterimsessiontab.cpp

```diff
@@ -433,16 +434,34 @@ void LLFloaterIMSessionTab::draw()
 
 void LLFloaterIMSessionTab::enableDisableCallBtn()
 {
-    if (LLVoiceClient::instanceExists() && mVoiceButton)
+    if (!mVoiceButton)
+        return;
+
+    BOOL enable = FALSE;
```

Comment:
`bool`?

Reply:
This branch is still using BOOL

Reply:
if you pass `bool` value as an argument is should work as well, right?

Reply:
release/webrtc-voice is kinda a dead branch.  Development has been moved to atlasaurus, and after to develop.

Reply:
> Development has been moved to atlasaurus, and after to develop.

Retargeted to develop

Reply:
nice, so there shouldn't be any fake bools there